bpf/bpf_host.c

```diff
@@ -1651,7 +1651,23 @@ int cil_to_host(struct __ctx_buff *ctx)
 	 * to mark as PACKET_OTHERHOST and drop.
 	 */
 	ctx_change_type(ctx, PACKET_HOST);
-#endif
+
+# ifdef ENABLE_NODEPORT
+	if (!validate_ethertype(ctx, &proto)) {
+		ret = DROP_UNSUPPORTED_L2;
```

Comment:
Do we need to actually drop this traffic? Or can we just skip the RevDNAT for it?